numpy/core/numeric.py

```diff
@@ -2023,7 +2024,7 @@ def warn_if_insufficient(width, binwidth):
         binary = bin(num)[2:]
         binwidth = len(binary)
         outwidth = (binwidth if width is None
-                    else max(binwidth, width))
+                    else builtins.max(binwidth, width))
```

Comment:
Hope we haven't missed any of these.

Reply:
that's why we have tests I'd say:) more seriously, this is set up as spaghetti. `numeric` importing from `fromnumeric`, tweaking `__module__`, and then another special-case import in `core/__init__.py` for `max` makes it hard to untangle. maybe I'll really bite that bullet at some point.